x/participationrewards/keeper/callbacks.go

```diff
@@ -89,5 +85,9 @@ func RewardsCallback(k Keeper, ctx sdk.Context, response []byte, query icqtypes.
 		k.icsKeeper.SetIntent(ctx, zone, di, true)
 	}
 
+	// set zone ValidatorSelectionAllocation to zero
+	zone.ValidatorSelectionAllocation = sdk.NewCoins(sdk.NewCoin("uqck", sdk.ZeroInt()))
```

Comment:
whilst this always will be `uqck`, it's probably correct to fetch `k.BondDenom(ctx)` from the staking keeper.